### ISSUE TITLE
Accessibility violation: Missing Definition List

### ISSUE THREAD

**Issue by pafech** [swiss/designsystem#155]
Webrichtlinien Bund sollten barrierefrei sein.
Das ist unmöglich, wenn kein Definition List Element angeboten wird.
Im Storybook/Source Code keine Erwähnung gefunden.

Wann werden die Webrichtlinien barrierefrei oder ist das gar kein Ziel?
